internal/extensions/extension.go

```diff
@@ -0,0 +1,4 @@
+package extensions
```

Comment:
The package is currently in plural form. Do we want it in singular instead? "extensions" vs "extension"

Reply:
Will fix in final pr